toolbox/build/build.sh

```diff
@@ -10,7 +10,6 @@ ANSIBLE_PYTHON=python3.8
 
 dnf -y update
 dnf -y install \
-    ansible \
```

Comment:
This won't break CI, but it will break the Vagrant provisioning, won't it?

Reply:
it is still installed later in the 

```
if [ "${NO_VAGRANT:-0}" != "1" ]; then
```

clause

Reply:
:+1: thanks